src/PathUtilities.cpp

```diff
@@ -108,14 +107,13 @@ namespace
 
     /// @brief Removes all needless instances of "." or ".." and makes appropriate edits to the provided path.
     /// @param ToRemove The path to remove dot segments from.
-    /// @param Separator The directory separator for splitting and rebuilding the path.
+    /// @param DirSep The directory separator(s) for splitting and rebuilding the path.
```

Comment:
Any reason not to Spell out "DirectorySeparator"

Reply:
120 character limit per line.  At least that's usually the reason when I shorten names.